usql Windows binary not found (404)

When running `make build-extension` on my Mac Intel (amd64), the build fails with the following error:

```
 => CANCELED [client-builder 5/7] RUN --mount=type=cache,target=/usr/src/  0.7s
------
 > [dl 6/6] RUN <<EOT ash:
#25 0.675 curl: (22) The requested URL returned error: 404
#25 0.681 tar: /out/windows/usql.tar.bz2: Cannot open: No such file or directory
#25 0.681 tar: Error is not recoverable: exiting now
#25 0.683 chmod: /out/windows/usql: No such file or directory
------
executor failed running [/bin/sh -c <<EOT ash
    if [ "amd64" = "$TARGETARCH" ]; then
        mkdir -p /out/windows
        curl -fSsLo /out/windows/usql.tar.bz2 "${USQL_RELEASE_URL}-windows-${TARGETARCH}.tar.bz2"
        tar -xf /out/windows/usql.tar.bz2 -C /out/windows
        chmod a+x /out/windows/usql
    fi
EOT]: exit code: 1
make: *** [build-extension] Error 1
```

It seems the URL for the usql Windows binary is not correct: it points to `"${USQL_RELEASE_URL}-windows-${TARGETARCH}.tar.bz2"` and it should be `"${USQL_RELEASE_URL}-windows-${TARGETARCH}.zip"` instead.

--- a/Dockerfile
+++ b/Dockerfile
@@ -16,7 +16,7 @@ RUN npm run build
 
 FROM alpine AS dl
 WORKDIR /tmp
-RUN apk add --no-cache curl tar
+RUN apk add --no-cache curl tar unzip
 ARG TARGETARCH
 ARG USQL_VERSION=0.13.4
 ARG USQL_RELEASE_URL="https://github.com/xo/usql/releases/download/v${USQL_VERSION}/usql-${USQL_VERSION}"
@@ -37,9 +37,9 @@ EOT
 RUN <<EOT ash
     if [ "amd64" = "$TARGETARCH" ]; then
         mkdir -p /out/windows
-        curl -fSsLo /out/windows/usql.tar.bz2 "${USQL_RELEASE_URL}-windows-${TARGETARCH}.tar.bz2"
-        tar -xf /out/windows/usql.tar.bz2 -C /out/windows
-        chmod a+x /out/windows/usql
+        curl -fSsLo /out/windows/usql.zip "${USQL_RELEASE_URL}-windows-${TARGETARCH}.zip"
+        unzip /out/windows/usql.zip -d /out/windows
+        chmod a+x /out/windows/usql.exe
     fi
 EOT
 

